x/btcstaking/keeper/msg_server.go

```diff
@@ -234,7 +236,7 @@ func (ms msgServer) CreateBTCDelegation(goCtx context.Context, req *types.MsgCre
 	}
 
 	// add this BTC delegation, and emit corresponding events
-	if err := ms.AddBTCDelegation(ctx, newBTCDel); err != nil {
+	if err := ms.AddBTCDelegation(ctx, newBTCDel, paramsValidationResult.MinUnbondingTime); err != nil {
 		panic(fmt.Errorf("failed to add BTC delegation that has passed verification: %w", err))
 	}
```

Comment:
Tbh it's a bit cleaner to not have the `MinUnbondingTime` as a parameter in this function, but if we retrieve it again from DB inside `AddBTCDelegation` then it's more DB footprint :sweat_smile:

Reply:
agree it is a bit cleaner, but imo it is not that much cleaner to be worth two additional database calls 😅